pkgs/tools/misc/brltty/default.nix

```diff
@@ -13,10 +14,12 @@ stdenv.mkDerivation rec {
     sha256 = "14psxwlvgyi2fj1zh8rfykyjcjaya8xa7yg574bxd8y8n49n8hvb";
   };
 
-  nativeBuildInputs = [ pkg-config python3.pkgs.cython tcl ];
-  buildInputs = [ bluez ]
+  nativeBuildInputs = [ pkg-config python3.pkgs.cython tcl ]
+    ++ lib.optional espeak-ngSupport espeak-ng;
+  buildInputs = [ bluez espeak-ng ]
```

Comment:
```suggestion
  buildInputs = [ bluez ]
```

Reply:
We want it in `buildInputs` if we want to use it.

Reply:
It is also added in line 22.

Reply:
Oh, nice, a triple whammy.